src/main/java/core/basesyntax/service/impl/ReadFromFileCsvImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadFromFileCsvImpl implements ReaderService {
+    private static final String FIRST_LINE_REPORT = "type,fruit,quantity";
+
+    public List<String> readFromFile(String filePath) {
+        List<String> readReport = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            String value = reader.readLine();
+            while (value != null) {
+                if (value.equals(FIRST_LINE_REPORT)) {
+                    value = reader.readLine();
+                }
```

Comment:
let's just skip the first line
we won't be able to reuse your class with a different csv file